powerlevel9k.zsh-theme

```diff
@@ -429,10 +429,12 @@ prompt_dir() {
   fi
 
   local current_icon=''
-  if [[ $(print -P "%~") == '~'* ]]; then
+  if [[ $(print -P "%~") == '~' ]]; then
     "$1_prompt_segment" "$0_HOME" "$2" "blue" "$DEFAULT_COLOR" "$current_path" 'HOME_ICON'
-  else
+  elif [[ $(print -P "%~") == '~'* ]]; then
     "$1_prompt_segment" "$0_DEFAULT" "$2" "blue" "$DEFAULT_COLOR" "$current_path" 'FOLDER_ICON'
```

Comment:
This is the second state. So the icon should be the new `HOME_SUB_ICON` and you need to push that state (something like `$0_HOME_SUBFOLDER` instead of `$0_DEFAULT`) to `$1_prompt_segment`.